account_invoice_supplierinfo_update/README.rst

```diff
@@ -0,0 +1,82 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================================
+Update Supplier Info of product from Supplier Invoice
+=====================================================
+
+This module allows in the supplier invoice, to automatically update all
```

Comment:
"This module allows to automatically update all products information in vendor bill for which the purchase information on the line are different from the supplier information defined in the product form."